include/SPTPersistentCache/SPTPersistentCache.h

```diff
@@ -227,16 +231,25 @@ typedef NSString * _Nonnull(^SPTPersistentCacheChooseKeyCallback)(NSArray<NSStri
 - (void)unscheduleGarbageCollector;
 /**
  * Delete all files files in managed folder unconditionaly.
+ * @param callback May be nil if not interested in result.
+ * @param queue Queue on which to run the callback. If callback is nil this is ignored otherwise mustn't be nil.
  */
-- (void)prune;
+- (void)pruneWithCallback:(SPTPersistentCacheResponseCallback _Nullable)callback
```

Comment:
How about keeping `- (void)prune` as a convenience method, which would also decrease the amount of API breakage.